src/sidebar/test/annotation-fixtures.js

```diff
@@ -6,6 +6,7 @@
 function defaultAnnotation() {
   return {
     id: 'deadbeef',
+    created: '2015-05-10T20:18:56.613388+00:00',
```

Comment:
Note this change to a long-standing fixture as the `created` date is suddenly much more important.